tests/tests/deployment.rs

```diff
@@ -2,23 +2,27 @@ use fuel_core::chain_config::ChainConfig;
 use fuel_core_types::fuel_tx::GasCosts;
 
 #[test_case::test_case(
-    include_str!("../../deployment/scripts/chainspec/beta_chainspec.json"); 
+    "./../deployment/scripts/chainspec/beta";
     "Beta chainconfig"
 )]
 #[test_case::test_case(
-    include_str!("../../deployment/scripts/chainspec/dev_chainspec.json"); 
-    "Dev chainconfig"
+    "./../deployment/scripts/chainspec/dev";
+    "Dev chainconfig" 
 )]
-fn test_deployment_chainconfig(bytes: &str) {
-    let mut chain_config: ChainConfig =
-        serde_json::from_str(bytes).expect("Should be able to decode chain config");
+fn test_deployment_chainconfig(path: &str) {
+    let chain_config = ChainConfig::load_from_directory(path)
+        .expect("Should be able to load chain config");
+
+    // TODO: add state config check in subsequent PR
```

Comment:
Could you create an issue to track it?=)

Reply:
This TODO will be merged into the subsequent regenesis PRs and will be addressed there with an updated test before it hits the master branch. Maybe we can skip opening an issue in this case?